diff -ruNb a/cblocks/Makefile.in b/cblocks/Makefile.in
--- a/cblocks/Makefile.in	2013-04-14 05:38:10.000000000 +0100
+++ b/cblocks/Makefile.in	2013-05-30 18:32:35.391546259 +0100
@@ -33,7 +33,7 @@
 	install -c ./cblocks $(bindir)/cblocks
 	install -d $(mandir)/man6
 	install -c ./cblocks.6 $(mandir)/man6/cblocks.6
-	install -d -g games $(datadir)
+	install -d $(datadir)
 	install -c ./series/* $(datadir)/
 
 movelist.o: movelist.c gen.h movelist.h
diff -ruNb a/csokoban/Makefile.in b/csokoban/Makefile.in
--- a/csokoban/Makefile.in	2013-04-14 05:35:14.000000000 +0100
+++ b/csokoban/Makefile.in	2013-05-30 18:32:29.583411851 +0100
@@ -32,7 +32,7 @@
 	install -c ./csokoban $(bindir)/csokoban
 	install -d $(mandir)/man6
 	install -c ./csokoban.6 $(mandir)/man6/csokoban.6
-	install -d -g games $(datadir)
+	install -d $(datadir)
 	install -c ./series/* $(datadir)/
 
 movelist.o: movelist.c gen.h movelist.h
